cmd/extract.go

```diff
@@ -157,9 +157,8 @@ func extractValidateAndRun() error {
 			return printLeafItemSummary(kv, out)
 		} else if opts.printKey {
 			return printLeafItemKey(kv, out)
-		} else {
-			return printLeafItemValue(kv, outMediaType, out)
 		}
+		return printLeafItemValue(kv, outMediaType, out)
```

Comment:
@mmorel-35 this has been fixed in https://github.com/etcd-io/auger/pull/218/files